rebar.config

```diff
@@ -85,7 +85,7 @@
        ]}.
 
 {plugins, [{swagger_endpoints, {git, "https://github.com/aeternity/swagger_endpoints", {ref, "4cdc9e0ad0b68501fc9fba8ac0712e46c1f8541e"}}},
-           {rebar_aecuckooprebuilt_dep, {git, "https://github.com/aeternity/rebar3-cuckoo-prebuilt-plugin.git", {ref, "2b2f3b3cf969ee91ba41d8351f3808530a8bf28e"}}}
+           {rebar_aecuckooprebuilt_dep, {git, "https://github.com/aeternity/rebar3-cuckoo-prebuilt-plugin.git", {ref, "20da01ecc347e448f2ae6186975f677bcd2466e0"}}}
```

Comment:
```suggestion
           {rebar_aecuckooprebuilt_dep, {git, "https://github.com/aeternity/rebar3-cuckoo-prebuilt-plugin.git", {ref, "0eef4488973a568ed24da201a7b7c1df465d2406"}}}
```

Reply:
This would include https://github.com/aeternity/rebar3-cuckoo-prebuilt-plugin/pull/2

Reply:
Applied.